ios/Sources/Component/View/FeedContentListView.swift

```diff
@@ -0,0 +1,58 @@
+import Model
+import SwiftUI
+
+public struct FeedContentListView: View {
+    private var feedContents: [FeedContent]
+    private let tapContent: (FeedContent) -> Void
+    private let tapFavorite: (String) -> Void
```

Comment:
closure argument type should be `FeedContent` to divide calling `addFavorite` and `removeFavorite`

Reply:
There are too many place to fix about this.
Why not fix this when adding repositories to get connected with each screens.